app/controllers/application_controller.rb

```diff
@@ -1,19 +1,11 @@
 class ApplicationController < ActionController::Base
 
+  include Authenticate
   before_action :authenticate_user
+  helper_method(:current_user)
```

Comment:
przeniesienie samego 
```
  helper_method(:current_user)
```
powoduje błąd: undefined method `helper_method' for Authenticate:Module, więc dodałam metodę: 
```
  def self.included(location)
    location.helper_method :current_user
  end
```